pinot-query-planner/src/main/java/org/apache/pinot/query/planner/logical/StagePlanner.java

```diff
@@ -49,15 +40,10 @@
  * This class is non-threadsafe. Do not reuse the stage planner for multiple query plans.
  */
 public class StagePlanner {
-  private final PlannerContext _plannerContext;
   private final WorkerManager _workerManager;
-
-  private Map<Integer, StageNode> _queryStageMap;
-  private Map<Integer, StageMetadata> _stageMetadataMap;
   private int _stageIdCounter;
 
-  public StagePlanner(PlannerContext plannerContext, WorkerManager workerManager) {
-    _plannerContext = plannerContext;
```

Comment:
I know planner context is not yet used in this class. but let's don't do this cleanup here. 
the _plannerContext holds some query options that might be useful later. let's pass `_plannerContext` into the constructor of the `GenerateQueryPlan` visitor